Docs/Build-DSC.md

```diff
@@ -0,0 +1,60 @@
+## Table of Contents
+- [Setup DSC and OMI Repo](#setup-dsc-and-omi-repo)
+- [Build OMI](#build-omi)
+- [Build DSC](#build-dsc)
+- [Run OMI Server](#run-omi-server)
+- [Test DSC basic functionality](#test-dsc-basic-functionality)
+- [Update DSC and rebuild](#update-dsc-and-rebuild)
+
+
+## Setup DSC and OMI Repo
+
+1. Clone the repo. 
+   - git clone https://github.com/Microsoft/PowerShell-DSC-for-Linux.git 
+   - cd PowerShell-DSC-for-Linux
+2. mkdir omi-1.3.0-2 
+3. mkdir omi-1.0.8 
+4. Download the source code for this release - https://github.com/Microsoft/omi/releases/tag/v1.3.0-2 
```

Comment:
why can't we clone omi and build it. We just have to build omi once on the dev box and never do it again. And only build Dsc for new changes.

Reply:
We could. I was trying to get one approach where it works. To reiterate, this is not the only way. This is one way to unblock our team from building DSC.

Reply:
Even if you build using sources from zip file. Building omi every time, deleting omi server binaries, copying some binaries from one version to another does not tell us which omi we actually used to validate and test DSC changes.